Dockerfile

```diff
@@ -1 +1,16 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE="https://github.com/ArkCase/ark_solr" \
+      MAINTAINER="Armedia LLC"
+
+# install SASL for mail server authentication that uses TLS
+RUN yum update -y && \
+    yum install postfix cyrus-sasl-lib cyrus-sasl-plain -y && \
+    sed -i 's@^inet_interfaces = localhost@inet_interfaces = all@g' /etc/postfix/main.cf
+
+EXPOSE  587
+CMD ["/usr/libexec/postfix/master","-c", "/etc/postfix", "-d"]
```

Comment:
should we be using CMD or ENTRYPOINT?  @fabricetriboix ?

Reply:
@JonHolman We should always use `CMD`, unless we have a very good reason to use `ENTRYPOINT`
https://stackoverflow.com/questions/21553353/what-is-the-difference-between-cmd-and-entrypoint-in-a-dockerfile
In essence, modifying `ENTRYPOINT` can be used to customize the behaviour of an image to the outside world. For example, I believe the official mysql image uses `ENTRYPOINT` to run either as a MySQL server or as a MySQL client using the same image.